tests/ValueParsers/IsoTimestampParserTest.php

```diff
@@ -139,8 +145,8 @@ public function validInputProvider() {
 				TimeValue::PRECISION_DAY,
 
 			),
-			'+0000000000000001-01-14T00:00:00Z (Julian)' => array(
-				'+0000000000000001-01-14T00:00:00Z',
+			'+0000000000002001-01-14T00:00:00Z (Julian)' => array(
+				'+0000000000002001-01-14T00:00:00Z',
```

Comment:
This test case is supposed to test that the calendar given as part of the input is used. If the input is Year 1, Julian would be assumed anyway, so the test would be inconclusive. To check whether the explicit calendar is actually used, we have to take a year that isn't Julian per default.